test/integration/apis/issue.test.ts

```diff
@@ -72,7 +83,15 @@ describe("issue", () => {
             await setExchangeRate(385523187);
             const amount = api.createType("Balance", 100000);
             const requestResult = await issueAPI.request(amount);
-            assert.isTrue(requestResult.hash.length > 0);
+            assert.equal(requestResult.hash.length, 32);
+
+            const issueRequests = await issueAPI.list();
+            const thisRequest = issueRequests.find(request => request.hash === requestResult.hash);
+
+            assert.isDefined(thisRequest, "Could not find issue request");
+            if (thisRequest !== undefined) {
```

Comment:
Since we assert `thisRequest` is defined in the line above, we can remove the if statement